carbon-design-system/carbon
# fix(data-table): update menu to match visible and accessible labels

Closes #4861

Match visible and accessible labels in `TableToolbarMenu`

#### Changelog

**New**

**Changed**

- Have `iconDescription` match `ariaLabel`

**Removed**

#### Testing & Reviewing

- Open the `DataTable with toolbar` story and run DAP
- Confirm violation no longer occurs
